app/helpers/settings_helper.rb

```diff
@@ -20,7 +21,8 @@ def grouped_settings(settings)
   end
 
   def setting_to_hash(setting)
-    presenter = SettingPresenter.from_setting(setting)
+    presenter = Foreman.settings.find(setting.name)
+    logger.warn("Setting #{setting.name} doesn't exist anymore, clean up your database") and return nil if presenter.nil?
```

Comment:
I'm not a fan of the `and return nil`. I prefer to be explicit here:
```suggestion
    if presenter.nil?
      logger.warn("Setting #{setting.name} doesn't exist anymore, clean up your database")
      return nil
    end
```